code/modules/paperwork/ticketmachine.dm

```diff
@@ -167,6 +167,8 @@
 		user.adjust_fire_stacks(1)
 		user.IgniteMob()
 
+/obj/machinery/ticket_machine/attack_ai(mob/user)//AI cannot interact with ticket_machine
+	return
 /obj/item/ticket_machine_ticket
```

Comment:
Usually you want to put any procs *after* the object definition, so something like: 
```js
/obj/item/ticket_machine_ticket
    name = "Ticket"
    ... // other stuff here, description, etc.

/obj/machinery/ticket_machine/attack_ai(mob/user)//AI cannot interact with ticket_machine
	return
```

It just looks better that way. Also it's good to have a blank line between every proc or path

Reply:
Yep, for sure.

Reply:
Yeah but the method I added is for the machine, so it goes after the other machine methods, not near the ticket ones.

Reply:
You're right, I mixed up the two similar paths here. So the way you have it is fine. The blank line after the return would still be nice though.

Reply:
still has spacing issues.